core/pages/Category.js

```diff
@@ -115,6 +115,10 @@ export default {
   created () {
     this.$bus.$on('filter-changed-category', this.onFilterChanged)
     this.$bus.$on('list-change-sort', (param) => { this.onSortOrderChanged(param) })
+    if (config.priceTiers) {
+      this.$bus.$on('user-after-loggedin', this.refreshData)
```

Comment:
instead of relying on event bus can we just watch the property from vuex store as a computed one?

We will eb slowly removing event bus so it should be avoided

Reply:
For now, we need an event bus. During the second phase (tier price + qty) we can change it